server/mongodb/actions/Card.js

```diff
@@ -10,9 +10,25 @@ export async function createCard(card) {
 export async function updateCardById(id, updatedCard) {
   await mongoDB();
 
-  return Card.findOneAndUpdate({ _id: id }, updatedCard, {
-    returnDocument: "after",
-  });
+  const { isThumbsUp, isThumbsDown, ...cardUpdate } = updatedCard;
+
+  if (isThumbsUp) {
+    return Card.findOneAndUpdate(
+      { _id: id },
+      { $inc: { thumbsUp: 1 } },
```

Comment:
think about passing the 1 as a parameter instead of a constant. what happens if it's an unlike operation? it will be -1 instead of 1

Reply:
But wouldn't thumbs down be counting for the number of unlikes? so the value will always be >= 0?